task2.js

```diff
@@ -1,6 +1,16 @@
 const csvPath = './csv/node_mentoring_t1_2_input_example.csv';
 const csv = require('csvtojson');
+const fs = require('fs');
 
 csv().fromFile(csvPath).then((jsonObj) => {
-  console.log(jsonObj);
+//   console.log(jsonObj);
+//   console.log(typeof jsonObj);
+  if (jsonObj) {
+    jsonObj.forEach((value) => {
+      fs.appendFile('task2.txt', `${JSON.stringify(value)}\n`, (err) => {
```

Comment:
About the way you are using `fs.appendFile`

This is not a requirement of the homework, but I'll comment on it to raise you awareness of async functions;

The `forEach` loop is a synchronous loop, on which you start a asynchronous task.

The result is that all file writes are done in parallel and therefore the order of write is not guaranteed. Have a look at you `task2.txt`, it will probably have a different order than the CSV, and as an experiment, you can delete it and generate again to see the order probably changing again.

If you refactor the code to use streams, as suggested by the other comment, this problem will be fixed.

Reply:
I changed the way to write lines into txt file via stream